python-frontend/src/main/protobuf/symbols.proto

```diff
@@ -74,13 +74,14 @@ message ClassSymbol {
   repeated string super_classes = 4;
   repeated FunctionSymbol methods = 5;
   repeated OverloadedFunctionSymbol overloaded_methods = 6;
-  bool has_decorators = 7;
-  bool has_metaclass = 8;
-  bool is_enum = 9;
-  bool is_generic = 10;
-  bool is_protocol = 11;
-  optional string metaclass_name = 12;
-  repeated string valid_for = 13;
+  repeated VarSymbol vars = 7;
```

Comment:
I think it's a bit weird to call this `vars`. What about `attributes`?

Reply:
Indeed... I used `var` in `typeshed_serializer` because that allows to reuse the existing merge functions for both classes and modules through duck typing. I'd go for keeping that and then serialize to `attributes` on the protobuf side then, wdyt? Or would you prefer we have class attributes everywhere for consistency?

Reply:
ok for me, we can keep `var` in `typeshed_serializer` and just change it the protobuf